build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-python3 starfyre --dev=True --path="test-application/" && python -m starfyre --build=True --path="test-application/"
+python3 starfyre --build --path="my-first-app"
```

Comment:
We do not have a folder called `my-first-app` in our project. It should still be`test-application` for us.